spec/plans/action/install_spec.rb

```diff
@@ -0,0 +1,40 @@
+# spec/spec_helper.rb
+
+# Load the BoltSpec library
+require 'bolt_spec/plans'
+
+describe 'peadm::action::install' do
+  # Include the BoltSpec library functions
+  include BoltSpec::Plans
+
+  # Configure Puppet and Bolt before running any tests
+  before(:all) do
+    BoltSpec::Plans.init
+  end
+
+  # something needs done here for the function
+
+  it 'minimum variables to run' do
+    allow_task('peadm::precheck').return_for_targets(
+      'primary' => {
+        'hostname' => 'primary',
+        'platform' => 'el-7.11-x86_64'
+      }
+    )
+    expect_task('peadm::mkdir_p_file').be_called_times(4)
```

Comment:
I'm curious what is our rationale for writing plan unit tests. It seems that we need them to test logic and data manipulation, but if tasks and sub plans are executed unconditionally, I wonder whether it's effective to repeat the calling expectations sequentially in a unit test?

Reply:
So this ticket is intended as a starter to at least give some shape of a minimal run and check before we try to build some proper logic and contexts on the plans